src/SAWScript/Crucible/JVM/MethodSpecIR.hs

```diff
@@ -25,65 +25,57 @@ Stability   : provisional
 {-# LANGUAGE ViewPatterns #-}
 {-# OPTIONS_GHC -fno-warn-orphans  #-}
 
-module SAWScript.Crucible.JVM.MethodSpecIR where
+module SAWScript.Crucible.JVM.MethodSpecIR
+  ( JIdent
```

Comment:
Note to reviewers: this module still exports the same things that it did before this patch. I added an explicit export list here because this module now re-exports some things from `SAWScript.Crucible.JVM.Setup.Value`, which requires using an export list to accomplish.